package.json

```diff
@@ -87,6 +87,8 @@
   },
   "dependencies": {
     "lodash": "^4.17.21",
+    "react-native-paper": "^5.3.1",
+    "react-native-vector-icons": "^9.2.0",
```

Comment:
I know loading fonts can be tricky sometimes, is this handled by autolinking?

Reply:
No, in fact Paper links here: https://github.com/oblador/react-native-vector-icons#installation and those instructions have you disable autolinking for react-native-vector-icons.   It works, [PR is here](https://github.com/lifeomic/react-native-sdk/pull/77), but I'm not 100% that this is the best way.

Reply:
Interesting, this would mean a consumer would have to do this themselves if they set up their own repo or we would have to include it in the starter repo. I wonder if we should just ship with chroma icons instead? I am not sure we have chroma as font though.

Reply:
I think for the moment we should just include it with the starter repo.  It's actually not that complicated or problematic, I'm just surprised it is not more automated.

For the most part, I'm guessing consumers are going to want to use their own icons anyway.